type.proto

```diff
@@ -70,6 +70,8 @@ message Object {
   BoundingBox bbox = 4;
   Vector3 velocity = 5;
   TrackingStatus tracking_status = 6;
+  google.protobuf.Timestamp timestamp = 7;
```

Comment:
One question, what is difference of this timestamp and the timestamp in `OutputMessage`?
```
message OutputMessage {
  google.protobuf.Timestamp timestamp = 1;

Reply:
In the distributed system, objects from different algo nodes can be captured at different timestamps. We decided to put a timestamp on each object to inform when the object is observed.

Reply:
Hmm, this is the result after object merging phase. then the observing timestamp should not be different whether Sensr consist with distributed system or not.
And these first detected timestamp of the object is managed inside client side program in order to reduce network bandwidth.
Can I get the history of decision?
I'd like to remove this redundant data if there is not a big requirement.
(Some of customer use LTE router for sending these result, adding a single timestamp per object per frame can lead huge data consuming.)

Reply:
No, it's different. Let's say an algo node hasn't sent algo result for 2 seconds, then the objects in the region are observed 2 seconds ago so we can't say the objects are still there.

Reply:
Here's the corresponding JIRA issue :wink: https://seoulrobotics.atlassian.net/browse/AR-3947

Reply:
Thank you for the explanation.
so this is "TIME OFFSET FOR EACH OBJECTS to indicate the time difference of each object to the latest point cloud created"? 
Then, can we use `google.protobuf.Duration time_offset =7` ?
Or leave a comment what exactly `timestamp` mean here?

Reply:
Oh, good suggestion! :heart: I like time_offset more than timestamp because it's vague as you pointed out. Let me apply your suggestion. Do you think I should reserve the field number?

Reply:
No, you don't need to. `dev` branch is not the officially released branch.
You can re-use `7` here.